Gemfile

```diff
@@ -15,3 +15,5 @@ source 'https://rubygems.org'
 gem 'cocoapods', '1.11.3'
 gem 'cocoapods-generate', '2.0.1'
 gem 'danger', '8.4.5'
+
+# Global Tap
```

Comment:
@paulb777, could I get a second opinion on https://github.com/firebase/firebase-ios-sdk/actions/runs/3143495797/jobs/5108309795?

This failure wasn't surfaced in the relevant PR that added `FIRLogWarningSwift`. The Podfile used by the test does install FirebaseCoreExtension which exports the `FIRLogWarningSwift` API.

Apart from updating the `FIRLogger.[mh]`, was there anything else you did to get `FIRLogDebugSwift` working?

Reply:
Running locally, I got it to succeed... 🤔

Reply:
```shell
▸ Touching FirebasePodTest.app (in target 'FirebasePodTest' from project 'FirebasePodTest')
▸ Build Succeeded
+ [[ 0 == 65 ]]
+ [[ 0 != 0 ]]

```

Reply:
I reproduced locally

Reply:
Discussed offline. The failure is likely due the to the C function being used by an SDK FirestoreSwift  that depends on obj-c++/c++ code (Firestore). 

Addressing in https://github.com/firebase/firebase-ios-sdk/pull/10286